tox.ini

```diff
@@ -438,6 +439,7 @@ deps =
     requests213: requests>=2.13,<2.14
     requests218: requests>=2.18,<2.19
     requests219: requests>=2.19,<2.20
+    sanic: sanic
```

Comment:
Are there more versions we should be testing?

Reply:
Thanks Kyle, I have added more [sanic versions](https://github.com/DataDog/dd-trace-py/commit/1dc1d42b8d6bb042b646cf98d04f12d7c9ca268d) for testing.